scss/_utilities_text-align.scss

```diff
@@ -10,19 +10,22 @@
 @mixin vf-u-text-align--center {
   .u-text-align--center {
     text-align: center !important;
+    justify-content: center !important;
```

Comment:
@anthonydillon @barrymcgee I think if we're adding properties which are meant only for flex we should state that explicitly. I would have no clue that ```u-content-align``` would only get effected if used with in a flex content. 

I suggest we should keep the utility classes to fully explain what they're achieving e.g 

```u-text-align--left``` or ```u-text-align--justify``` and they both ONLY effect that. If justify content is for flex it should be a ```u-flex-content--center``` or something similar as trumps should only overwrite one property.